nbgrader/html/static/js/models.js

```diff
@@ -121,6 +121,9 @@ var CommentUI = Backbone.View.extend({
         this.listenTo(this.model, "sync", this.animateSaved);
 
         this.$comment.attr("placeholder", this.model.get("auto_comment") || "Comments");
+
+        autosize(document.querySelectorAll('textarea'));        
```

Comment:
I think you can use `this.$comment` to use just the specific textarea rather than querying all of them. This `initialize` function will get called once for each comment, so if you select them all here it'll end up doing n^2 resizes :-)